roles/lib_utils/action_plugins/master_check_paths_in_config.py

```diff
@@ -45,12 +46,19 @@ def pop_migrated_fields(mastercfg):
     """Some fields do not need to be searched because they will be migrated
     for users automatically"""
     # Walk down the tree and pop the specific item we migrate / don't care about
-    for item in ITEMS_TO_POP:
-        field = mastercfg
-        for sub_field in item:
-            parent_field = field
-            field = field[sub_field]
-        parent_field.pop(item[len(item) - 1])
+    for field_path in ITEMS_TO_POP:
+        pop_migrated_field(mastercfg, field_path)
```

Comment:
Not sure why would another function be required here - just readability? It doesn't look like its being used anywhere else

Reply:
Yeah, just readability and code elegance. Returning from a function struck me as a clean way to handle when a path wasn't found.